resolwe/flow/executors/docker/__init__.py

```diff
@@ -30,15 +36,56 @@ def start(self):
             'container_image': self.requirements.get('image', settings.FLOW_EXECUTOR['CONTAINER_IMAGE']),
         }
 
+        # Set resource limits.
+        limits = []
+        if 'cores' in self.resources:
+            # Each core is equivalent to 1024 CPU shares. The default for Docker containers
+            # is 1024 shares (we don't need to explicitly set that).
+            limits.append('--cpu-shares={}'.format(int(self.resources['cores']) * 1024))
```

Comment:
The problem with this is that currently all inputs are directly exposed as top-level elements in the context. We can't just add `requirements` in there as there could be an input called `requirements`?